src/app/shared/components/login-form/login-form.component.ts

```diff
@@ -13,12 +16,22 @@ export class LoginFormComponent {
   password: string = "";
   submitted: boolean = false;
 
+  constructor(
+    private authService: AuthService,
+    private userStoreService: UserStoreService
+  ) {}
+
   onLogin(): void {
     this.submitted = true;
     if (this.loginForm.valid) {
+      this.authService
+        .login(this.loginForm.value)
+        .pipe(map(() => this.userStoreService.getUser().subscribe()))
+        .subscribe();
```

Comment:
Consider checking if login failed. Also, I miss where the redirection is happening when logged in?